src/components/POLine/Item/ItemForm.js

```diff
@@ -1,128 +1,173 @@
-            name="receiving_note"
-          />
-        </Col>
-        <Col xs={6}>
-          <ContributorForm />
-        </Col>
-        <Col xs={6}>
-          <Field
-            backendDateStandard={DATE_FORMAT}
-            component={Datepicker}
-            dateFormat={DATE_FORMAT}
-            fullWidth
-            label={<FormattedMessage id="ui-orders.itemDetails.subscriptionFrom" />}
-            name="subscription_from"
-            timeZone={TIMEZONE}
-          />
-        </Col>
-        <Col xs={6}>
-          <Field
-            component={TextField}
-            fullWidth
-            label={<FormattedMessage id="ui-orders.itemDetails.publisher" />}
-            name="publisher"
-          />
-        </Col>
-        <Col xs={6}>
-          <Field
-            label={<FormattedMessage id="ui-orders.itemDetails.subscriptionInterval" />}
-            name="subscription_interval"
-            component={TextField}
-            type="number"
-            fullWidth
-          />
-        </Col>
-        <Col xs={6}>
-          <Field
-            backendDateStandard={DATE_FORMAT}
-            component={Datepicker}
-            dateFormat={DATE_FORMAT}
-            fullWidth
-            label={<FormattedMessage id="ui-orders.itemDetails.publicationDate" />}
-            name="publication_date"
-            timeZone={TIMEZONE}
-          />
-        </Col>
-        <Col xs={6}>
-          <Field
-            backendDateStandard={DATE_FORMAT}
-            component={Datepicker}
-            dateFormat={DATE_FORMAT}
-            fullWidth
-            label={<FormattedMessage id="ui-orders.itemDetails.subscriptionTo" />}
-            name="subscription_to"
-            timeZone={TIMEZONE}
-          />
-        </Col>
-        <Col xs={6}>
-          <Field
-            component={TextField}
-            fullWidth
-            label={<FormattedMessage id="ui-orders.itemDetails.edition" />}
-            name="edition"
-          />
-        </Col>
-        <Col xs={6}>
-          <MaterialTypesForm materialTypes={parentResources.materialTypes} />
-        </Col>
-        <Col xs={12}>
-          <ProductIdDetailsForm />
-        </Col>
-        <Col xs={12}>
-          <Field
-            component={TextArea}
-            fullWidth
-            label={<FormattedMessage id="ui-orders.itemDetails.description" />}
-            name="description"
-          />
-        </Col>
-      </Row>
+      <Fragment>
+        <Row style={{ display: 'flex', alignItems: 'flex-end', justifyContent: 'space-between' }}>
+          <Col xs={11}>
+            <Field
+              component={TextField}
+              fullWidth
+              label={<FormattedMessage id="ui-orders.itemDetails.title" />}
+              name="title"
+            />
+          </Col>
+          <Col xs={1}>
+            {this.selectInstanceModal()}
+          </Col>
+        </Row>
+        <Row>
```

Comment:
it doesn't make too much sense to me using `Row` like this. It will be the same if you just add `<Col xs={12}>`

Reply:
updated
![screenshot_6](https://user-images.githubusercontent.com/43407139/50596960-d6144480-0eb6-11e9-9a8d-2b0d1160ddd6.png)